### PR TITLE
Users rebuild preparation

### PR DESCRIPTION
**Pull request by anpolimus**
Code review fixes for Personify and Daxko.
@hamrant , please check code only.
We will setup separate instance for testing of Personify.

### CODE REVIEW

**Review by hamrant — APPROVED** [review]

**Comment by hamrant** [comment]
The code looks ok for me, there will be small conflicts with my PR

**Comment by jediandrey** [comment]
retest

File: js/gated-content/src/store/index.js [hunk 1]
```diff
@@ -30,10 +27,7 @@ export default new Vuex.Store({
     auth,
     // @TODO: It seems like only one provider can use recaptcha at a time.
     // This breaks `authCustom` and should be resolved before PR is merged.
-    authDaxkoBarcode,
     authCustom,
```
Comment (hamrant) [inline js/gated-content/src/store/index.js:30]: I think we can delete authCustom too

```
import authCustom from './modules/auth/custom';
```

File: src/Form/GCSettingsForm.php [hunk 2]
```diff
@@ -152,6 +152,9 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
   public function submitForm(array &$form, FormStateInterface $form_state) {
     $settings = $this->config('openy_gated_content.settings');
     $settings->setData($form_state->getValue('app_settings'));
+    $settings->set('event_add_to_calendar', $form_state->getValue('event_add_to_calendar'));
```
Comment (hamrant) [inline src/Form/GCSettingsForm.php:155]: Please remove these lines, no need to set params separately, they already in `app_settings`

**Review by hamrant — CHANGES_REQUESTED** [review]

**Comment by hamrant** [comment]
Events not available for dummy user: 
Screenshot from 2020-09-24 12-21-47

Dummy User can click on `My account` link in the main menu and see `/user` page:

Screenshot from 2020-09-24 12-24-27

And can access the user edit page:

Screenshot from 2020-09-24 12-24-37

I think we need to close access to these pages for virtual y roles too.

**Review by hamrant — APPROVED** [review]
@jediandrey please fix comments in separated PR

**Pull request merged**